.travis.yml

```diff
@@ -14,6 +14,9 @@ addons:
 
 before_install:
   - export JOBS=max
+  
+before_script:
+  - npm install typescript ts-node -g
```

Comment:
Is it possible to add these as `devDependencies` instead?

Reply:
done. :)